lib/Migration/Version31001Date20240210105322.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2022 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\FullTextSearch\Migration;
+
+use Closure;
+use Doctrine\DBAL\Types\Type;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\Attributes\AddIndex;
+use OCP\Migration\Attributes\IndexType;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+#[AddIndex('fulltextsearch_index', IndexType::INDEX, 'fix live index select')]
+class Version31001Date20240210105322 extends SimpleMigrationStep {
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if (!$schema->hasTable('fulltextsearch_index')) {
+			return null;
+		}
+
+		$table = $schema->getTable('fulltextsearch_index');
+		if ($table->hasIndex('fts_cse')) {
+			return null;
+		}
+
+		$table->addIndex(['collection', 'status', 'err'], 'fts_cse');
```

Comment:
Have you tried to write the index like this?
```suggestion
		$table->addIndex(['collection', 'err', 'status'], 'fts_ces');
```
Not sure of the cardinality, but `err` uses `=` and not `<>` so I guess it should be faster.
Another way is to rewrite the query and use a `status IN(<all other statuses>)` but it requires to have a few statuses